.github/workflows/uberjar.yml

```diff
@@ -33,7 +40,7 @@ jobs:
         distribution: 'temurin'
     - name: Install Clojure CLI
       run: |
-        curl -O https://download.clojure.org/install/linux-install-1.10.3.933.sh &&
+        curl -O https://download.clojure.org/install/linux-install-1.10.3.1058.sh &&
```

Comment:
IMO it's better to keep this at the same version (and look at the next line).
If we want to upgrade the required Clojure version, we should have a separate PR for that (and change various places to stay consistent).

Reply:
interesting, specially because it didn't fail!, just reverted that change (fully agree with you on versioning and keeping consistency)